client/homescreen/welcome-modal/style.scss

```diff
@@ -8,6 +8,11 @@
 
 		.components-button {
 			color: $white;
+
+			// @todo Remove this after https://github.com/WordPress/gutenberg/issues/28311 is fixed
+			span {
```

Comment:
I'd love it if this was a more specific selector, but given that this only affects the welcome modal and there are no other classes specific to this, I think this looks okay 👍